pkg/config/ext_authz_test.go

```diff
@@ -222,6 +253,41 @@ func Test_externalAuthZFilter_extAuthz(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "http with pack as bytes option",
```

Comment:
nit: It seems to me that this should be "http with path prefix with pack bytes enabled".